CITATION.cff

```diff
@@ -0,0 +1,34 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: ' FenicsXConcrete '
```

Comment:
Are the spaces around the name there for a reason?
```suggestion
title: 'FenicsXConcrete'
```
I would suggest to expand the title a bit. For example:
"FenicsXConcrete - structural FE simulations", or something along those lines that fit, just to give a little bit more context.

Reply:
The spaces were inserted by the tool that I used to fill the file. I have asked myself the same question, but I guessed that they know what they are doing.
How is tthat for a short title?

_FenicsXConcrete -- Structural simulations of Concrete in FEniCSx_ ?